lib/utils/branches.js

```diff
@@ -1,5 +1,6 @@
 var _ = require('lodash');
 
+var Normalize = require('./normalize');
```

Comment:
`normalize` instead of `Normalize`, as far as I can see. It is not a class constructor.